test/main.spec.ts

```diff
@@ -30,6 +31,13 @@ beforeEach(() => {
     });
 });
 
+jest.mock('@actions/core', () => ({
```

Comment:
@yahavi this mock is mandatory for the tests to work.
I tried to do it at each test case like i did in line 36, but for some reason it doesn't work.